### ISSUE TITLE
tpm2-tss fails in warrior

### ISSUE THREAD

**Issue by emilm** [jiazhang0/meta-secure-core#135]
**automake: error: cannot open < aminclude_static.am: No such file or directory**

I am using warrior branch in yocto, and master branch in meta-secure-core.

This is fixed in tpm2-tss https://github.com/tpm2-software/tpm2-tss/pull/1256 
Issue: https://github.com/tpm2-software/tpm2-tools/issues/1291

patch: https://github.com/tpm2-software/tpm2-tss/pull/1256/files/4403f3bb15bd233d5d52a8e09ada71716c9a0465

So all you need to do is to add "AX_ADD_AM_MACRO_STATIC([])" to https://github.com/jiazhang0/meta-secure-core/blob/master/meta-tpm2/recipes-tpm/tpm2-tss/tpm2-tss/0001-build-update-for-ax_code_coverage.m4-version-2019.01.patch#L47 

Or am I way off here?

@lumag

**Comment by ksaye** [comment]
@emilm, were you able to get this to work?  I am struggling with the same error on warrior.

**Comment by emilm** [comment]
@ksaye Yes, sorta, I moved on to meta-security instead, which works with TPM2, even the virtual TPM2 device ! https://git.yoctoproject.org/cgit/cgit.cgi/meta-security

**Comment by 2pack94** [comment]
@ksaye if you still want to build it, I hacked the file "0001-build-update-for-ax_code_coverage.m4-version-2019.01.patch" to include the change that emilm proposed. This fixed the error for me. See the new file here:
[fixed_patch.txt](https://github.com/jiazhang0/meta-secure-core/files/4103319/fixed_patch.txt)

**Comment by ckrzen** [comment]
Fixed, for me, while configuring tpm2-tss-2.3.2 by installing ``autoconf-archive`` from here: http://gnu.mirrors.pair.com/autoconf-archive/?C=M;O=D

SOURCE: https://github.com/openbmc/sdbusplus/issues/29

**Comment by kamomil** [comment]
Hi,
I get the same issue with ubuntu 18.04:
```
# :~/git/tpm2-tss$ ./bootstrap 
Generating file lists: src_vars.mk
configure.ac:28: installing './compile'
configure.ac:23: installing './config.guess'
configure.ac:23: installing './config.sub'
configure.ac:12: installing './install-sh'
configure.ac:12: installing './missing'
automake: error: cannot open < aminclude_static.am: No such file or directory
autoreconf: automake failed with exit status: 1
# :~/git/tpm2-tss$ ./bootstrap
Generating file lists: src_vars.mk
configure.ac:21: error: possibly undefined macro: AC_SUBST
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:63: error: possibly undefined macro: AS_IF
configure.ac:96: error: possibly undefined macro: AC_MSG_ERROR
configure.ac:125: error: possibly undefined macro: AC_MSG_WARN
autoreconf: /usr/bin/autoconf failed with exit status: 1
# :~/git/tpm2-tss$ ./bootstrap
Generating file lists: src_vars.mk
automake: error: cannot open < aminclude_static.am: No such file or directory
autoreconf: automake failed with exit status: 1
```

**Comment by kamomil** [comment]
solved for me after installing `autoconf-archive`
